mlflow/tracking/fluent.py

```diff
@@ -379,6 +656,26 @@ def delete_run(run_id):
     Deletes a run with the given ID.
 
     :param run_id: Unique identifier for the run to delete.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Set existing run_ids to delete
+        run_ids = ["13ee9e661cbf4095a7c92cc55b4e12b4", "948fbf2d0b7f4056b3dd4914845a1e1b"]
+
+        # Delete run_ids and fetch the results.
+        # Note that runs are not actually delete, only lifecycle stage is set to "deleted"
+        [mlflow.delete_run(run_id) for run_id in run_ids]
```

Comment:
```suggestion
        for run_id in run_ids:
            mlflow.delete_run(run_id)
```

I think using list comprehension like this is not a good practice.

Reply:
While list comps are Pythonic, elegant, sometimes more efficient than for loops, here we are not building a list, nor are we
filtering or mapping, so it's not appropriate to use, for this particular case, so will change it elsewhere too